### ISSUE TITLE
405 Method not allowed - Playnite

### ISSUE THREAD

**Issue by HairyHook** [Lacro59/playnite-howlongtobeat-plugin#170]
Hi Lacro59,

I'm getting a 405 error code when authenticating:
![image](https://user-images.githubusercontent.com/63165721/198782715-d0170694-ff05-4294-b34e-961ac565b51a.png)
Any idea how to fix it? Am I the only one seeing this?

TIA

**Comment by stevietv** [comment]
same here when trying to authenticate

**Comment by TheDarkSpock** [comment]
Same thing is also happening to me

**Comment by TheJasonPlayz** [comment]
Yeah! I can't get any of the games to load, and when I try to authenticate, it either lets me login and just doesn't do anything or gives me this error.

**Comment by SisterVersio** [comment]
Same thing

**Comment by barrelltitor** [comment]
Same issue here

**Comment by barrelltitor** [comment]
Same here, but I found a solution. Click the login button in the top right. I guess the login URL is not correct?

**Comment by Lacro59** [comment]
Can you try with the last release?

**Comment by barrelltitor** [comment]
Latest version fixed it. It also seems not to want me to login again every day
